src/librustc/ty/constness.rs

```diff
@@ -83,15 +88,36 @@ impl<'tcx> TyCtxt<'tcx> {
         }
 
         if self.features().staged_api {
```

Comment:
The function this is in is called `is_min_const_fn`, but the body comments make it sounds more like "is the fn required to conform to min_const_fn restrictions", which is not reflected in the function name. Maybe `must_be_min_const_fn`?

Reply:
well... it's both :D We call `is_min_const_fn` to check whether a function can be called from other min const fn. When checking a function we call `is_min_const_fn` to check if we are callable from other min const fn and thus must conform.